### PR TITLE
Updating the ui to look better

### PR DESCRIPTION
**Pull request by vooxal**
This isn't done, im just putting it out there for now

### CODE REVIEW

**Comment by vooxal** [comment]
Oh god this isn't how its supposed to end! do i need to create another fork, ;-;

File: src/components/AddButton.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const ButtonContainer = styled.fieldset`
 	flex-direction: column;
 	border: none;
 	padding: none;
-	background-color: white;
+	background-color: transparant;
```
Comment (getkey) [inline src/components/AddButton.tsx:20]:
```suggestion
	background-color: transparent;
```

File: src/components/ModeBar.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const RadioGroup = styled.fieldset`
 `;
 
 const Label = styled.label`
-	opacity: 0.1;
+	background-color:darkcyan;
```
Comment (getkey) [inline src/components/ModeBar.tsx:26]:
```suggestion
	background-color: darkcyan;
```

**Comment by vooxal** [comment]
im dumb so im just going to delete it and re-fork it and then change it again

**Pull request closed without merging**